qiskit/circuit/add_control.py

```diff
@@ -151,15 +177,25 @@ def control(operation, num_ctrl_qubits=1, label=None):
     return cgate
 
 
+def _toggle_ctrl_state(qc: QuantumCircuit,
+                       num_qubits: int,
+                       ctrl_state: int) -> Gate:
+    if isinstance(ctrl_state, int) and 0 <= ctrl_state < 2**num_qubits:
```

Comment:
The first condition seems redundant as "int" is what the function signature expects anyway.

Reply:
ok